src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -74,42 +74,37 @@ public void RawSendInCaseOfFailure(IModel channel, string address, byte[] body,
 
         public void Initialize(IModel channel, string mainQueue)
         {
-
             CreateExchange(channel, mainQueue);
             channel.QueueBind(mainQueue, mainQueue, string.Empty);
-
         }
 
-        static string ExchangeName(Type type)
-        {
-            return type.Namespace + ":" + type.Name;
-        }
+        static string ExchangeName(Type type) => type.Namespace + ":" + type.Name;
```

Comment:
@SimonCropp in this case there would be no issue so we could indeed use string interpolation.

The slightly unsettling aspect for me is that in C# 5 we'd never use current culture for things like this. In moving to C# 6, in .NET 4.6 we can simply swap all our current concatenations and invariant `string.Format`s for bling strings wrapped in `Invariant()`, but in .NET 4.5 we're forced to make a decision about current vs invariant culture in each case, since bling string = current culture.

It would be less risky to stay away from bling strings in .NET 4.5 but if we feel the benefit of the syntax outweighs the risk then I'm happy to go with that.